src/Microsoft.AspNetCore.Mvc.ViewFeatures/IViewComponentHelper.cs

```diff
@@ -16,7 +16,14 @@ public interface IViewComponentHelper
         /// Invokes a view component with the specified <paramref name="name"/>.
         /// </summary>
         /// <param name="name">The name of the view component.</param>
-        /// <param name="arguments">Arguments to be passed to the invoked view component method.</param>
+        /// <param name="arguments">
+        /// Arguments to be passed to the invoked view component method.
```

Comment:
This sentence now just repeats the "Arguments to be passed to the invoked view component method." words. Remove it.